### PR TITLE
episerver has this feature remaining Url, it is ignored by this packag…

### PR DESCRIPTION
**Pull request by gatisb**
…e #52

### CODE REVIEW

**Comment by valdisiljuconoks** [comment]
@marisks any comments?

File: src/Geta.EPi.Extensions/LinkItemExtensions.cs [hunk 1]
```diff
@@ -15,26 +15,26 @@ public static class LinkItemExtensions
...
+                ? content.ContentLink
+                : ContentReference.EmptyReference;
         }
 
         /// <summary>
         ///     Returns IContent for provided LinkItem if it is EPiServer content otherwise returns null.
         /// </summary>
         /// <param name="source">Source LinkItem for which to return content.</param>
-        /// <returns>Returns IContent for provided LinkItem if it is EPiServer content otherwise returns null.</returns>
+        /// <returns>Returns IContent for provided LinkItem if it is EPiServer content otherwise returns null. Note that remainingUrl part will be lost use with caution.</returns>
```
Comment (marisks) [inline src/Geta.EPi.Extensions/LinkItemExtensions.cs]: As it returns content, the comment is wrong. Content doesn't have remaining URL ;)

File: src/Geta.EPi.Extensions/LinkItemExtensions.cs [hunk 2]
```diff
@@ -15,26 +15,26 @@ public static class LinkItemExtensions
         ///     Returns ContentReference for provided LinkItem if it is EPiServer content otherwise returns EmptyReference.
         /// </summary>
         /// <param name="source">Source LinkItem for which to return content reference.</param>
-        /// <returns>Returns ContentReference for provided LinkItem if it is EPiServer content otherwise returns EmptyReference.</returns>
+        /// <returns>Returns ContentReference for provided LinkItem if it is EPiServer content otherwise returns EmptyReference. Note that remainingUrl part will be lost use with caution.</returns>
```
Comment (marisks) [inline src/Geta.EPi.Extensions/LinkItemExtensions.cs]: `ContentReference` doesn't have remaining URL. So the comment is not needed.

File: src/Geta.EPi.Extensions/LinkItemExtensions.cs [hunk 3]
```diff
@@ -45,11 +45,22 @@ public static IContent ToContent(this LinkItem source)
         /// <returns>Returns friendly URL if item is EPiServer content, otherwise returns the original Href property value.</returns>
         public static string GetFriendlyUrl(this LinkItem linkItem, bool includeHost = false)
         {
-            var contentReference = linkItem.ToContentReference();
+            if (linkItem.Href == null)
```
Comment (marisks) [inline src/Geta.EPi.Extensions/LinkItemExtensions.cs]: Why not checking `string.IsNullOrEmpty'? Is empty string a valid value to proceed?

**Pull request merged**